src/tests/setupTests.ts

```diff
@@ -1,5 +1,8 @@
 // enables assertions such as toBeInTheDocument to be used in our tests
 import '@testing-library/jest-dom';
+import { TextEncoder, TextDecoder } from 'util';
+
+Object.assign(global, { TextDecoder, TextEncoder });
```

Comment:
Out of curiosity (not a nit) what's this doing?

Reply:
This was something that used to be available with `jest-dom`, but after updating it needs to be added manually.